### DISCUSSION TITLE
Adding a fully fledged marketplace on website

### DISCUSSION THREAD

**Discussion by reinart1** [chromiecraft/chromiecraft#6494] [Suggestions]
Marketplace on website where you could sell BoE's, characters and gold to other players for chromiepoints. 

Positives

1) would keep the people engaged to sell gold and BoE's until they have enough chromie points to fully deck out their toons with cosmetic items
2) players would have a way to get chromiepoints other than donations (obviously the points would have to be purchased by someone else to keep the circulation going)
3) if player gets to 70 and gets bored of their geared character, but dont like the idea of leveling a new character and gearing it up, they could just sell their old one and buy a new one from someone else
4) has a high chance for donations to rise and the profits could be reinvested back into the server ^^

Negatives

1) people with a lot of money and new to the server could just buy someone elses decked out character. (player engagement would still be high because the one who is selling is most probably just spending the coins for QoL features or for buying a new toon to keep on playing)
2) has methods for corruption within if not done correctly

All in all it would be a similiar system to Warmanes marketplace and seeing their success I think a lot of it is thanks to their amazing website marketplace which is another system to keep players engaged.

P.S such system should have taxes implemented against inflation to keep points valuable

**Comment by 55Honey** [comment]
Hello reinart1,

thanks for your suggestion.
Such a market place is the absolute opposite of our standards and breaks one of our most important rules:

```
1.12) No offers or actual trades for any kind of in-game service, item, or currency for real-world goods.
```

ChromiePoints, since they're obtainable for money, being the real-world goods.
In short: This is not going to happen.

Regards

**Comment by reinart1** [comment]
Huge loss in the potential what the server could become then sadly as the time has shown that most servers without a website marketplace will eventually wither away.

**Reply by 55Honey** [reply]
I appreciate your concerns, but we're around for 3 years now and we plan to stay, don't worry :-)

**Reply by reinart1** [reply]
Yea im glad and the amazing job you do on azerothcore is commendable, but on the business perspective and player engagement to have a concurrent player count more than 500 then such a system would greatly contribute to. If you fear P2W accusations then its really not a P2W as the goods are traded from player to player (server would only provide QoL features) and it would 100% prolong the server and make coding the azerothcore even more effective and faster if the donations are invested back into it. Just food for thought.

Ty for attention.
